db/psychologists.ts

```diff
@@ -145,14 +170,11 @@ const upsertMany = async (psyList: Psychologist[]): Promise<void> => {
     }
   });
 
+  await Promise.all(upsertArray);
+
   if (psychologistsToInsert.length > 0) {
-    upsertArray.push(
-      db(psychologistsTable)
-    .insert(psychologistsToInsert),
-    );
+    await db(psychologistsTable).insert(psychologistsToInsert);
```

Comment:
I don't understand why you remove the insert from the array ?

Reply:
Because we need to get the coordinates first and then insert them. We can't do both in the same `Promise.all`. We can call if needed.